src/iam.ts

```diff
@@ -1677,4 +1561,135 @@ export class IAM extends IAMBase {
     }
     throw new Error(ERROR_MESSAGES.CACHE_CLIENT_NOT_PROVIDED);
   }
+
+  protected async validateChangeOwnership({
+                                            namespaces,
+                                            newOwner
```

Comment:
Minor point but I feel like this indenting is a bit odd. Is this your preferred style?

Reply:
I will revert that one. I use default from webstorm.